qframe.go

```diff
@@ -1102,16 +1103,18 @@ func (qf QFrame) ToCSV(writer io.Writer) error {
 	for _, s := range qf.columns {
 		row = append(row, s.name)
 	}
-
 	columns := make([]column.Column, 0, len(qf.columns))
 	for _, name := range row {
 		columns = append(columns, qf.columnsByName[name])
 	}
 
 	w := stdcsv.NewWriter(writer)
-	err := w.Write(row)
-	if err != nil {
-		return err
+	var err error
```

Comment:
Please make the err variable local to the if-block below and initialize a new err in the loop below to reduce the scope of the variables. The previous implementation simply reused the existing top level err variable to avoid having to come up with a new name or get lint warnings about shadowing the outer err variable.

Reply:
I'll have a look, thank you.

Reply:
Agreed

Reply:
Done